src/main/java/net/minecraftforge/common/data/ForgeBlockTagsProvider.java

```diff
@@ -74,21 +75,22 @@ public void registerTags()
         func_240522_a_(GRAVEL).func_240534_a_(Blocks.GRAVEL);
         func_240522_a_(NETHERRACK).func_240534_a_(Blocks.NETHERRACK);
         func_240522_a_(OBSIDIAN).func_240534_a_(Blocks.OBSIDIAN);
-        func_240522_a_(ORES).addTags(ORES_COAL, ORES_DIAMOND, ORES_EMERALD, ORES_GOLD, ORES_IRON, ORES_LAPIS, ORES_REDSTONE, ORES_QUARTZ);
+        func_240522_a_(ORES).addTags(ORES_COAL, ORES_DIAMOND, ORES_EMERALD, ORES_GOLD, ORES_IRON, ORES_LAPIS, ORES_REDSTONE, ORES_QUARTZ, ORES_NETHERITE_SCRAP);
         func_240522_a_(ORES_COAL).func_240534_a_(Blocks.COAL_ORE);
         func_240522_a_(ORES_DIAMOND).func_240534_a_(Blocks.DIAMOND_ORE);
         func_240522_a_(ORES_EMERALD).func_240534_a_(Blocks.EMERALD_ORE);
-        func_240522_a_(ORES_GOLD).func_240534_a_(Blocks.GOLD_ORE);
+        func_240522_a_(ORES_GOLD).func_240531_a_(BlockTags.field_232866_P_);
```

Comment:
Should we keep this? It would make things more consistent as all ores have a tag under forge namespace, but it's also just a copy of the vanilla tag

Reply:
I am not sure as well, I just think it is just a tad easier, avoids having to add to a minecraft tag only for gold ore.

Reply:
We should keep all of our tags. Having ours extend from vanilla's by default.
It keeps things organized, and it makes things not just dissapear in the middle of a MC version.
We could possibly make it deprecated. But honestly it's a data object it doesn't really matter how long we keep it.